mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,23 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
 );
 
 router.put(
   '/order/:orderId/status/:userId',
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
+);
+
+router.delete(
+  '/order/:orderId/:userId',
+  requireSignin,
+  isAuth,
+  isAdmin,
+  removeOrder
```

Comment:
The controller function name in the route function for removing an order should be `removeOrder`, not `remove`. It should be `router.delete('/order/:orderId/:userId', requireSignin, isAuth, isAdmin, removeOrder);`.